### DISCUSSION TITLE
Home connect command

### DISCUSSION THREAD

**Discussion by housedomoticz** [alexkn/node-red-contrib-homeconnect#81] [General]
there is no command menu inside the home connect node.
How can I use the two pause and resume commands?

**Comment by ericstaal** [comment]
I got the same problem with a washer.
The commands option would be the way to trigger this. 

According the API 
/homeappliances/{haId}/commands
/homeappliances/{haId}/commands/{commandKey}

The command allowed may change based on the state of the device

**Reply by housedomoticz** [reply]
For my washer machine
I tested the command using http request in Node-Red. the debug result is the following

error: object
description: "The requested resource could not be found but may be available again in the future."
key: "404"

**Comment by ericstaal** [comment]
I got it working; will make a PR when it is all tested.

I have no right to create a PR :(

**Reply by alexkn** [reply]
Everyone should be able to create a PR.
You need to fork the repository, commit your changes there and the create a pullrequest from your branch.

**Reply by ericstaal** [reply]
Done: https://github.com/alexkn/node-red-contrib-homeconnect/pull/82

(i used the wrong pat..... :) )

**Reply by housedomoticz** [reply]
I have now tested the modified node. it works with my washer.thanks
